.github/ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,55 @@
-Make sure you are running the latest version of Home Assistant before reporting an issue.
-
-You should only file an issue if you found a bug. Feature and enhancement requests should go in [the Feature Requests section](https://community.home-assistant.io/c/feature-requests) of our community forum:
+<!-- READ THIS FIRST:
+- Make sure you are running the latest version of Home Assistant before reporting an issue: https://github.com/home-assistant/home-assistant/releases
+- Do not report issues for components if you are using custom components: files in <config-dir>/custom_components
+- Frontend (user interface of Home Assistant in your browser) related issues have to be reported here: https://github.com/home-assistant/home-assistant-polymer/issues
+- This is for bugs only. Feature and enhancement requests should go in our community forum: https://community.home-assistant.io/c/feature-requests
+- Answer all question as detailed as possible, write the information inside the backticks. Do not delete any text from this template!
+
+-->
+**1. Home Assistant version:**
+```
 
-**Home Assistant release (`hass --version`):**
+```
+<!--
+- Frontend -> developer tools -> info
+- Or use this command: hass --version
+-->
 
+**2.a) I run Hass.io or the Docker image**:
+```
 
-**Python release (`python3 --version`):**
+```
+<!--
+- Yes / No
+-->
 
+**2.b) ...No, I run an installation with this Python version:**
+```
 
-**Component/platform:**
+```
+<!--
+- Minimum supported version is Python 3.5.3
+- Use this command: python3 --version
+-->
 
+**3. Component/platform:**
+```
 
-**Description of problem:**
+```
+<!--
+- example: components/updater.py
```

Comment:
We should not ask people to post the python file, few people will know what to do here. Rather just make them post a link to the documentation.

Reply:
Having them link to the docs of the component that is not working will also allow us to automatically add labels to issues 👍